src/ontology/mondo-ingest.Makefile

```diff
@@ -471,10 +499,12 @@ slurp-all: $(foreach n,$(ALL_COMPONENT_IDS), slurp-$(n))
 ######### Analysis ##########
 #############################
 .PHONY: mapped-deprecated-terms
-mapped-deprecated-terms: mapped-deprecated-terms-artefacts mapped-deprecated-terms-docs
+mapped-deprecated-terms: mapped-deprecated-terms-artefacts
```

Comment:
Intended to have `mapped-deprecated-terms-docs` being removed?

Reply:
Eek, I feel bad for making mistakes like this. I meant to actually have it be an alias for `-docs`, not `-artefacts`, because the way that I set up the dependency structure, things were running twice.

Maybe you have some suggestions. It's not too complex, but you might have a better way to set it up.

```makefile
.PHONY: mapped-deprecated-terms
mapped-deprecated-terms: mapped-deprecated-terms-docs

.PHONY: mapped-deprecated-terms-docs
mapped-deprecated-terms-docs: ../../docs/reports/mapped_deprecated.md

../../docs/reports/mapped_deprecated.md: mapped-deprecated-terms-artefacts
	python3 $(SCRIPTSDIR)/deprecated_in_mondo.py --docs

.PHONY: mapped-deprecated-terms-artefacts
mapped-deprecated-terms-artefacts: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n)_mapped_deprecated_terms.robot.template.tsv)
```